checkov/terraform/checks_infra/solvers/attribute_solvers/contains_attribute_solver.py

```diff
@@ -0,0 +1,13 @@
+from checkov.terraform.checks_infra.solvers.attribute_solvers.base_attribute_solver import BaseAttributeSolver
+
+
+class ContainsAttributeSolver(BaseAttributeSolver):
+    operator = 'contains'
```

Comment:
let's add the names of all the operators to an enum? it makes it easy to track on what operators we have